src/Discovery/DiscoveryDetails/DataDownloadList/DataDownloadList.tsx

```diff
@@ -33,7 +36,7 @@ const DataDownloadList = ({
         healLoginNeeded={healLoginNeeded}
         noData={noData}
       />
-      {!noData && (
+      {!noData && showList && (
```

Comment:
Can you explain a bit why we need this `showList` variable here in addition to `noData`?

Reply:
This could ensure against a study having a valid  resource field value but is missing the correct data after the data is processed. I can update the variable name from _showList_ to _resourceFieldValueIsValid_ which would make more sense

Reply:
hmm okay I thin i get it now, thanks

Reply:
This has been updated: the variable _showList_ has been renamed in the _DataDownloadList_ component to _resourceFieldValueIsValid_ to improve clarity.